README.md

```diff
@@ -44,6 +50,37 @@ You will first need to setup your local environment and ensure that all configur
 
 ## <a name="contributing-your-code"></a> Contributing Your Code
 
+Are you willing to contribute to this project? You can contribute in many areas but primarily in the following areas
+
+1. Implementing endpoints and controllers
+2. Writing unit tests for endpoints and controllers. 
+3. Documentation 
+4. Creating middleware and their consumables
+5. Fixing/pointing out bugs
+6. We could use a boost in our code coverage, so any tests to cover untested fucntions is highly welcome
+
+### Quick Reference
+
+- All responses must follow the format specified in the online [swagger documentation](https://comments-microservice.herokuapp.com/). To facilitate this a few utility functions have been provided and you should use them to send your responses to stay consistent. 
+   + **responseHandler** for sending responses with custom response code, optional data and a message 
+   + **customError** for error responses with custom error response code, message and optional data. To use this in a try..catch block, create a new instance of the customError class with the above properties. Pass this new error object to the `next` callback in your controller znd that is it. An **errorHandler** utility will send the necessary response
+
+- A validation middleware is created to validate all endpoints using the `Joi` validation package. Validation rules are available for every endpoint and you must use a validation middleware when implementing an endpoint. If the validation rule/middleware for the endpoint you are not working on has not been implemented then contact the maintainers. 
```

Comment:
I don't think that they should contact us as maintainers. Rather encourage them to request assignment to a validation rule task that adds validation rules for the endpoint.

Reply:
I left this open to allow persons actually request the task but I'll update it to encourage them to open an issue for it and reach out if they do not understand